go.mod

```diff
@@ -12,3 +12,5 @@ require (
 	k8s.io/klog v1.0.0
 	k8s.io/utils v0.0.0-20200324210504-a9aa75ae1b89
 )
+
+replace github.com/aquasecurity/starboard => /Users/dpacak/go/src/github.com/aquasecurity/starboard
```

Comment:
We need remove this line once https://github.com/aquasecurity/starboard/pull/74 is merged and released